keyboards/hineybush/h60/info.json

```diff
@@ -0,0 +1,18 @@
+{
+    "keyboard_name": "h60",
+    "url": "",
+    "maintainer": "hineybush",
+    "width": 3,
+    "height": 2,
+    "layouts": {
+        "LAYOUT": {
```

Comment:
This needs updating.

Reply:
I'll just remove this file as VIA does the job better

Reply:
Except we require it for configurator and *really* should be added as part of this PR.

Its not the issue that via does end user configuration better, but that its part of adding keyboard support to QMK.